### PR TITLE
Created tutorial on how to setup a game server with AMP

### PR DESCRIPTION
**Pull request by ntimo**
Tutorial on how to setup a Game server with AMP from Qubecoders.

I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Timo 0mfhniozkb9s4q7e6ap8yvlt@nowitzki.me

### CODE REVIEW

File: tutorials/game-server-with-amp/01.en.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+You should now see a new instance in the `local instances` block. It should have the colour red, click onto it and then select `Start Instance`. Now when you click on the `Manage` button you will see the AMP interface for that instance and be able to manage it.
+
+## Step 7 - Make AMP start at boot
+
+In order to make AMP start at boot you will have to type `crontab -e` then choose a editor for example `nano`, then when you scroll all the way down you have to add `@reboot /opt/cubecoders/amp/ampinstmgr -b` to the file. You can then exit and save by pressing `control+x` if you are asked if want to save press enter to confirm.
+
+## Step 8 - Upgrade AMP
+
+Sometimes comes the time where a new AMP version is release, in order to upgrade AMP the latest version simply run `ampinstmgr upgradeall`
```
Comment (n-se) [inline tutorials/game-server-with-amp/01.en.md]:
```suggestion
Sometimes a new AMP version gets released. In order to upgrade AMP to  the latest version, simply run `ampinstmgr upgradeall`.
```

File: tutorials/game-server-with-amp/01.en.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/game-server-with-amp"
+slug: "minecraft-server-with-amp"
+date: "2019-03-17"
+title: "Game server server with AMP"
+short_description: "In this tutorial we are going to setup a game server server with AMP"
+tags: [, "Gaming", "Webinterface"]
+author: "ntimo"
+author_img: "https://avatars3.githubusercontent.com/u/6145026"
+---
```
[the remark this answers is not in the document]
Comment (ntimo) [reply tutorials/game-server-with-amp/01.en.md]: Going to readd them :)

File: tutorials/game-server-with-amp/01.en.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Step 2.1 - Install Java (for Minecraft servers)
+
+If you want to run a Minecraft server you will need to install java you can do so by executing:  
+`apt install -y openjdk-8-jre`
+
+## Step 3 - Install AMP
+
+Now you want to install AMP, in order to do connect to your server via ssh and then execute:  
+`sudo su -c "bash <(wget -qO- https://cubecoders.com/getamp.sh)"`
```
[the remark this answers is not in the document]
Comment (ntimo) [reply tutorials/game-server-with-amp/01.en.md]: Yes I took it from the official docs. But I am going to change it. This makes much more sense :)

**Comment by ntimo** [comment]
Thank you @n-se for the review :)

**Comment by Billito00** [comment]
Can you also add the new language tags from the Template File and the license block, please?

`language: "en"`
`available_languages: ["en"]`

**Comment by ntimo** [comment]
@Billito00 Done :)

File: tutorials/game-server-with-amp/01.en.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/game-server-with-amp"
+slug: "minecraft-server-with-amp"
+date: "2019-03-17"
+title: "Game server server with AMP"
+short_description: "In this tutorial we are going to setup a game server server with AMP"
+tags: [, "Gaming", "Webinterface"]
+author: "ntimo"
+author_link: "https://github.com/ntimo
```
Comment (n-se) [inline tutorials/game-server-with-amp/01.en.md]:
```suggestion
author_link: "https://github.com/ntimo"
```

File: tutorials/game-server-with-amp/01.en.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/game-server-with-amp"
+slug: "minecraft-server-with-amp"
+date: "2019-03-17"
+title: "Game server server with AMP"
+short_description: "In this tutorial we are going to setup a game server server with AMP"
+tags: [, "Gaming", "Webinterface"]
```
Comment (n-se) [inline tutorials/game-server-with-amp/01.en.md]:
```suggestion
tags: ["Gaming", "Webinterface"]
```

**Comment by leojakeson** [comment]
Please add a (short) conclusion, and the license, then we can merge this.

**Pull request merged**